### ISSUE TITLE
mouth animation missing

### ISSUE THREAD

**Issue by SwiftyWang** [YuganshT79/Smiley-Rating#2]
This is a cool project, but from the demo there should have a mouth animation. :)

**Comment by YuganshT79** [comment]
Yes, Implementing the mouth animation is quite trickt but it's in my TODO
tasks.

I will definitely implement the mouth animation in future. 😁
